ansible_collections/arista/avd/roles/eos_designs/python_modules/network_services/ethernet_interfaces.py

```diff
@@ -123,13 +123,13 @@ def ethernet_interfaces(self) -> list | None:
                                 if not vrf.get("_evpn_l3_multicast_enabled"):
                                     raise AristaAvdError(
                                         f"'pim: enabled' set on l3_interface {interface_name} on {self.shared_utils.hostname} requires evpn_l3_multicast:"
```

Comment:
let's add single quotes around `interface_name` and `self.shared_utils.hostname` as well for consistency

Reply:
Fixed, thanks!